clients/drcachesim/CMakeLists.txt

```diff
@@ -965,6 +965,23 @@ if (BUILD_TESTS)
       use_DynamoRIO_extension(tool.drcacheoff.burst_traceopts drcovlib_static)
     endif (NOT RISCV64)
 
+    if (NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
+      set(source_abs "${CMAKE_CURRENT_SOURCE_DIR}/tests/window_test.cpp")
```

Comment:
No need for a separate var for this as it's just used once.

Reply:
Done. Removed it.